bucket/tfsec.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "0.46.0",
+    "description": "tfsec uses static analysis of your terraform templates to spot potential security issues. Now with terraform v0.12+ support.",
+    "homepage": "https://github.com/aquasecurity/tfsec",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/aquasecurity/tfsec/releases/download/v0.46.2/tfsec-windows-amd64.exe",
```

Comment:
Looks like there is a version mismatch between v0.46.2 and the main version on line 2

Reply:
Good find. Fixed.